packages/infrastructure/mongodb_atlas_identity_provider/README.md

```diff
@@ -0,0 +1,96 @@
+import MarkdownAlert from "@/components/markdown/MarkdownAlert.astro";
+
+# Mongodb Atlas MongoDB Identity Provider
+
+This module sets up the identity provider configuration in MongoDB Atlas.
+
+<MarkdownAlert severity="warning">
+  Due to limitations with MongoDB Atlas, this module does not handle automatic de-provisioning of users. 
+  - Once Bypass SAML Mode is disabled, users who are no longer in the identity provider will be unable to log in.
+  - However, any application keys that the user has previously generated will still be active and continue to have access.
+  - Proper key rotation and revocation steps should be taken to ensure security.
+</MarkdownAlert>
+
+## Guide
+
+This guide will help setup the MongoDB Atlas side of the SSO integration.
+
+### Pre-req: Setup MongoDB Atlas Identity Provider Domain Verification
+
+1. Go to your MongoDB Atlas account
+2. Go to `Organization Settings` -> `Federated Authentication Settings` -> `Domains`
+3. Click on `Add Domain`
+4. Add the root domain that you are using for Authentik (ie: panfactum.com)
+5. Select `DNS Record` as the verification method
+6. Note the `TXT Record` that is generated
+
+### Deploy a new DNS TXT Record & Verify
+
+1. Add a text record to your `global/aws_dns/terragrunt.hcl` file that looks like [this](https://github.com/Panfactum/stack/blob/__PANFACTUM_VERSION_MAIN__/packages/reference/environments/production/global/aws_dns_records/terragrunt.hcl#118).
+2. Run `terragrunt apply`
+3. Go back to MongoDB Atlas and click on `Verify` next to the domain you added
+4. Confirm verification
+
+### Download the Signing Certificate from Authentik
+
+Before proceeding, make sure to have downloaded the signing certificate from the Authentik application
+1. Login to your Authentik instance
+2. Switch over to the `Admin` section
+3. Go to the section `Applications` -> `Providers` -> `MongoDB Atlas`
+4. Find `Related objects` and click on the `Download` button for the `Download signing certificate`
+
+### Deploy the MongoDB Identity Provider module
+
+Unfortunately, the terraform provider for MongoDB Atlas does not support the creation of the Identity Provider but allows for modifications. 
+We will first create the resource through the UI and then import it to configure further.
+
+From MongoDB Atlas UI
+1. Go to `Organization Settings`
+2. Note the `Organization ID`
```

Comment:
Anytime you specify "Note", you need to specifically say "You will need this later." otherwise "Note" is ambiguous.